infobip-typescript-generator-extension-annotation-processor-showcase/pom.xml

```diff
@@ -24,4 +24,12 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
This is added to copy resources to class/decorators structure but there is problem.
This does not work when you run mvn clean install
Becuse clean command deletes everything from target, and then starts AnnotationProcessor, processor starts implementations of typescript generators, and during processing codes fails because there are no decorators in target folder (clean command deleted content).

Reply:
old comment logic is now changed